crates/chaincash_server/src/api.rs

```diff
@@ -49,13 +50,18 @@ impl IntoResponse for ApiError {
         let (status_code, msg) = match self {
             ApiError::OffChain(e) => (e.as_status_code(), e.to_string()),
         };
-        let body = Json(json!({
+        let body = Body::from(json!({
```

Comment:
Sorry, I meant when using `axum::http::StatusCode` we can keep the rest of this how it was like:

```rust
//...
        let body = Json(json!({
            "error": {
                "detail": msg,
            }
        }));

        (status_code, body).into_response()
```